front-end/src/components/Home/index.js

```diff
@@ -1,58 +0,0 @@
-import React from 'react';
```

Comment:
We don't need this boilerplate create-react-app page anymore, as we're displaying the login page instead